src/app/register/register.component.html

```diff
@@ -35,10 +40,11 @@
           {{'GENDER_REQUIRED' | translate }}</p>
       </div>
 
+
       <div class="mb-3 pb-10">
         <label for="aadharId">{{'AADHAR_NO' | translate}}<span class="text-danger">*</span></label>
         <input type="text" class="form-control p-4" id="aadharId" name="aadharId" formControlName="aadharId"
-          placeholder="{{'ENTER_AADHAAR_NO' | translate}}">
+          minlength="12" maxlength="12" placeholder="{{'ENTER_AADHAAR_NO' | translate}}">
```

Comment:
@shreyas1434shinde comment the aadhar validation for now

Reply:
@itsvick comment resolved